docs/cache/develop/integrations/sequelize.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 6
+sidebar_label: Sequelize
+title: Integrate Momento With Sequelize ORM
+description: Supercharge your sequelize backed databases with Momento cache
+---
+
+# Integrating Momento with your Sequelize backed database
+
+[Sequelize](https://sequelize.org/) is a popular ORM for Oracle, Postgres, MySQL, MariaDB, SQLite and SQL Server, and more.
+
+In this tutorial, we'll walk through our [Momento sequelize model cache wrapper](https://github.com/momentohq/momento-sequelize-cache). By the end of the exercise, you'll be able to seamlessly cache your sequelize-backed ORM databases with minimal code changes.
+
+## Prerequisites
+
+To get this example running, you'll need to have the following available:
+
+- An account with a Git provider such as [GitHub](https://github.com/), [GitLab](https://gitlab.com), or [Bitbucket](https://bitbucket.org/).
+- Node version 16 and above installed on your machine https://nodejs.org/en/download
+
+Now you're ready to configure the Momento side of things using the [Momento console](https://console.gomomento.com). You can create an account in the console by providing your email address or linking an existing Google or GitHub account. Once you've logged into the console, click the `Create Cache` button on the top right of the page:
+
+![Create Cache button](/img/console-create-cache.png)
+
+Create a cache called `model-cache`. You can create the cache in your choice of cloud provider, AWS or GCP, and in any region.
+
+![Create cache form](/img/console-create-cache-model-cache.png)
+
+After pressing the `Create` button you'll see the new `model-cache` cache in the list of available caches.
+
+Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure that you choose the same region when you generate a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, choose the cloud provider and region you used to create your cache, choose the `Super User Token` token type, and hit the `Generate Token` button.
+
+![Generate token](/img/console-generate-api-key.png)
+
+Copy the `Api Key` and save it in a safe place. You'll need to use it later to run your application, where you will add it as an environment variable.
```

Comment:
Have added an image for this in this PR. We need to do a sweep to update the screenshots to the new API_KEY one.

Reply:
ohhh good catch. would you mind making a ticket for this?

Reply:
Sure

Reply:
https://github.com/momentohq/public-dev-docs/issues/501